ext/drutil/drutil.c

```diff
@@ -511,8 +505,10 @@ drutil_insert_get_mem_addr_arm(void *drcontext, instrlist_t *bb, instr_t *where,
                                           opnd_create_reg(base), opnd_create_reg(index),
                                           OPND_CREATE_INT(extend),
                                           OPND_CREATE_INT(amount));
-#    endif /* ARM/AARCH64 */
             PRE(bb, where, instr);
+#    else  /* RISCV64 */
+            ASSERT(false, "unreachable");
```

Comment:
Maybe add in the string sthg about there not being any base reg + index reg addressing mode in RISC-V?